.github/workflows/compile-examples.yml

```diff
@@ -26,11 +26,13 @@ jobs:
         - name: Arduino_DebugUtils
         - name: ArduinoMqttClient
         - name: Arduino_SecureElement
+        - name: Blues Wireless Notecard
       # sketch paths to compile (recursive) for all boards
       UNIVERSAL_SKETCH_PATHS: |
         - examples/ArduinoIoTCloud-Advanced
         - examples/ArduinoIoTCloud-Basic
         - examples/ArduinoIoTCloud-Callbacks
+        - examples/ArduinoIoTCloud-Notecard
```

Comment:
@zak better to remove the Notecard example from here and move it to the specific platform section so we can skip ESP8266 build

Reply:
I will make this change as you suggest, but I'm reaching out to the Cloud team for help on this matter as well.